nsv13/code/modules/overmap/FTL/components/drive.dm

```diff
@@ -71,6 +71,9 @@
 	pylons = null
 	return ..()
 
+/obj/machinery/computer/ship/ftl_core/swarmer_act()
+	return FALSE
```

Comment:
Can we please print an IC message here as to why.
Just copy paste from the other system critical devices.